contrib/guix/manifest.scm

```diff
@@ -71,9 +57,10 @@ http://www.linuxfromscratch.org/hlfs/view/development/chapter05/gcc-pass1.html"
                  (("-rpath=") "-rpath-link="))
                #t))))))))
 
-(define building-on (string-append (list-ref (string-split (%current-system) #\-) 0) "-guix-linux-gnu"))
+(define building-on (string-append "--build=" (list-ref (string-split (%current-system) #\-) 0) "-guix-linux-gnu"))
 
 (define (explicit-cross-configure package)
+  (define building-on (string-append (list-ref (string-split (%current-system) #\-) 0) "-guix-linux-gnu"))
```

Comment:
Why does this variable need to be redefined?
Can't this just be:
```diff
-  (define building-on (string-append (list-ref (string-split (%current-system) #\-) 0) "-guix-linux-gnu"))
-  (package-with-extra-configure-variable package "--build" building-on))
+  (package-with-extra-configure-variable package building-on))
```

Reply:
`package-with-extra-configure-variable` expects 3 arguments:
`(define (package-with-extra-configure-variable p variable value`
so we need a separate `--build` & value.

Reply:
Yeah totally, sorry for the noise.